service/history/replication/task_ack_manager.go

```diff
@@ -52,6 +53,8 @@ var (
 	errUnknownQueueTask       = errors.New("unknown task type")
 	errUnknownReplicationTask = errors.New("unknown replication task")
 	defaultHistoryPageSize    = 1000
+	minReadTaskSize           = 20
+	maxReplicationLatency     = int64(40)
```

Comment:
I suggest adding time unit suffix (I guess `Seconds`) or even better use time.Duration instead. As of now it is not clear what 40 means.